### ISSUE TITLE
AI considers Welcomebot a hostile mob

### ISSUE THREAD

**Issue by D3athrow-Issues** [d3athrow/vgstation13#14654]
(WEB REPORT BY: gunosaguki REMOTE: 172.93.109.202:7777)
Example: Nanotrasen minebot will autofire on a wearer of the welcomebot mask on sight. And the Welcomebot's HP is so low that itll gib you outright.

As funny as this is, I'm not so certain it is intentional

**Comment by Shadowmech88** [comment]
Doesn't the Nanotrasen minebot fire at *any* mob that isn't in the Neutral faction?
The Welcomebot is in the Spessmart faction, I believe.

**Comment by Gunosaguki** [comment]
Yeah I wasnt sure. regardless, it is an insta-kill due to the low HP of the welcomebot form which doesnt actually give any benefits beyond hiding your identity

**Comment by SkowronX** [comment]
new_vegas_faction_armor.jpg

**Comment by Shadowmech88** [comment]
That's the dice you roll with amorphous masks. Each one turns you into a random simple animal, so sometimes it's a not-so-good one.

**Comment by Exxion** [comment]
This seems more like an issue with the minebot than the mask
